map/generate-offline-map-with-local-basemap/README.md

```diff
@@ -21,9 +21,11 @@ The author of a web map can support the use of basemaps which are already on a d
 ## How to use the sample
 
 1. Click on the "Take Map Offline" button.
-2. You will be prompted to choose whether you wish to use the "naperville_imagery.tpk" basemap (already saved in the samples-data directory) or to download the online basemap.
-3. If you choose to download the online basemap, the offline map will be generated with the same (topographic) basemap as the online web map. To download the Esri basemap, you may be prompted to sign in to ArcGIS.com.
-4. If you choose to use the basemap from the device, the offline map will be generated with the local imagery basemap i.e. no tiles are exported or downloaded. Since the application is not exporting online ArcGIS Online basemaps you will not need to log-in.
+2. You will be prompted to choose whether you wish to download the online basemap or use the "naperville_imagery.tpk" basemap (which is already saved in the samples-data directory).
```

Comment:
Looks like the old one was an exception, but we usually don't have a numbered list in this section, just paragraphs.

Reply:
yeh we should turn this into a paragraph rather than numbered points. must have also been overlooked in the common sample revamp.